docs/reference/changelog.md

```diff
@@ -2,12 +2,77 @@
 
 [[toc]]
 
+## n8n@0.75.0
+For a comprehensive list of changes, check out the [commits](https://github.com/n8n-io/n8n/compare/n8n@0.74.0...n8n@0.75.0) for this version.
+
+- New nodes
+    - Box
+    - Box Trigger
+    - CrateDB
+    - Jira Trigger
+- Enhanced nodes
+    - Pipedrive
+    - GitLab
+    - NextCloud
+- Production/Test Webhooks now support OPTIONS request
```

Comment:
@tanay1337 - would love your feedback about this.

Reply:
This can be rephrased as, 'Webhooks now support OPTIONS request.'